fdbclient/BulkDumping.cpp

```diff
@@ -20,7 +20,37 @@
 
 #include "fdbclient/BulkDumping.h"
 
-BulkDumpState newBulkDumpTaskLocalSST(const KeyRange& range, const std::string& remoteRoot) {
+std::string stringRemovePrefix(std::string str, const std::string& prefix) {
+	if (str.compare(0, prefix.length(), prefix) == 0) {
+		str.erase(0, prefix.length());
+	} else {
+		return "";
+	}
+	return str;
+}
+
+Key getKeyFromHexString(const std::string& rawString) {
+	if (rawString.empty()) {
+		return Key();
+	}
+	std::vector<uint8_t> byteList;
+	ASSERT((rawString.size() + 1) % 3 == 0);
+	for (size_t i = 0; i < rawString.size(); i += 3) {
+		std::string byteString = rawString.substr(i, 2);
```

Comment:
What's format of hex here? Characters look like xff or something?  If so, is this subset right?  Should it be i+1?